blocks/hero-banner/hero-banner.css

```diff
@@ -213,4 +239,27 @@
         width: 63.75rem;
         margin-bottom: -2rem;
     }
+
+    .hero-banner .hero-banner-text-container::before {
+        content: none;
+    }
+
+    .hero-banner.overlap .hero-banner-heading-container {
+        padding-left: calc(50vw - 82.875rem / 2);
```

Comment:
Did you actually mean `calc(50vw - (82.875rem / 2))`? If so then you can I guess just take half of the value.

I guess you meant `calc((50vw - 82.875rem) / 2)` but don't you need extra braces for that?

Reply:
@bosschaert thanks for the comment. No, it keep the same specification as original site which shows a extra space in the left-hand side for PC view. It changes automatically depends on the width of the screen.
<img width="962" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/473200b3-fed1-46fd-8f20-f26e155b2397">

Reply:
Ah ok, so it really is just `calc(50vw - 41.4375rem)` I guess?